src/index.ts

```diff
@@ -39,17 +67,94 @@ async function run(): Promise<void> {
 
     await Deploy.getDeployment(appSelector, namespaceArg);
 
-    let route = await Deploy.getRoute(appName, namespaceArg);
     // To make it appear as a URL
-    route = `http://${route}`;
+    const route = `http://${await Deploy.getRoute(appName, namespaceArg)}`;
     ghCore.info(`✅ ${appName} is exposed at ${route}`);
 
     ghCore.setOutput(Outputs.ROUTE, route);
     ghCore.setOutput(Outputs.SELECTOR, appSelector);
 }
 
+async function createPullSecretFromAuthFile(createPullSecretFrom: string): Promise<void> {
+    if (createPullSecretFrom === "docker") {
+        isPullSecretCreated = await createPullSecretFromDocker();
```

Comment:
doesn't it make more sense to have this function return boolean instead of setting a global variable?